Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN apt-get update
 RUN apt-get -y install bash git cmake python3 python3-pip python3-virtualenv python3-cryptography python3-future python3-click python3-serial python3-wheel python3-pyparsing python3-pyelftools
 RUN ln -s /usr/bin/python3 /usr/bin/python
 
-RUN git clone --recursive https://github.com/espressif/ESP8266_RTOS_SDK.git --branch v3.4
+RUN git clone --recursive https://github.com/espressif/ESP8266_RTOS_SDK.git
+RUN cd /ESP8266_RTOS_SDK && git checkout a192988b7906680440213eefc730d99dec0ce237
```

Comment:
I wonder if we should provide some comment about what commit it is or why we pick this one 😇

Reply:
It's in the commit message :wink: